homework/index.js

```diff
@@ -0,0 +1,74 @@
+let repositories;
+async function getPromise() {
+  const res = await fetch('https://api.github.com/orgs/HackYourFuture/repos?per_page=100');
+  return res.json();
+}
+
+const myPromise = getPromise(); // Set the promise as a global variable
+
+async function selectedRepoCaracteristics(selectValue) {
+  const result = await myPromise;
+  result.forEach((x) => {
+    if (x.name === selectValue) {
+      document.getElementById('repoCaracteristics').innerHTML = `
+         <b>Repository :</b>   <a href="">${x.name} </a><br>
+          <b>Descripion :</b>   ${x.description} <br>
+          <b>Forks :</b>   ${x.forks} <br>
+          <b>Updated :</b>   ${new Date(x.updated_at).toLocaleString('en-US')} <br>`;
+    }
+  });
+}
+
+
+function showContributors(list) {
```

Comment:
Maybe call `list` '`contributors`'? Note: in the next part of the homework, you'll convert these into actual `Contributor` objects though ;).